workflows/microbiome/gene-based-pathogen-identification/Gene-based-Pathogen-Identification.ga

```diff
@@ -0,0 +1,1165 @@
+            "tool_state": "{\"input_collection\": {\"__class__\": \"ConnectedValue\"}, \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "0.0.2",
+            "type": "tool",
+            "uuid": "11aa5d81-81ba-4c37-89cd-5774c41d3e63",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": "extracted_lables",
+                    "output_name": "output",
+                    "uuid": "95421305-232a-47da-8c6d-9ac949b8efcf"
+                }
+            ]
+        },
+        "2": {
+            "annotation": "",
+            "content_id": "__BUILD_LIST__",
+            "errors": null,
+            "id": 2,
+            "input_connections": {
+                "datasets_0|input": {
+                    "id": 0,
+                    "output_name": "output"
+                }
+            },
+            "inputs": [],
+            "label": null,
+            "name": "Build list",
+            "outputs": [
+                {
+                    "name": "output",
+                    "type": "input"
+                }
+            ],
+            "position": {
+                "left": 180,
+                "top": 558.9999771118164
+            },
+            "post_job_actions": {
+                "RenameDatasetActionoutput": {
+                    "action_arguments": {
+                        "newname": "list_of_lists_of_processed_samples"
+                    },
+                    "action_type": "RenameDatasetAction",
+                    "output_name": "output"
+                }
+            },
+            "tool_id": "__BUILD_LIST__",
+            "tool_state": "{\"datasets\": [{\"__index__\": 0, \"input\": {\"__class__\": \"ConnectedValue\"}, \"id_cond\": {\"id_select\": \"idx\", \"__current_case__\": 0}}], \"__page__\": null, \"__rerun_remap_job_id__\": null}",
+            "tool_version": "1.2.0",
+            "type": "tool",
+            "uuid": "6e8d5663-2192-4a18-a383-7517f2cd739e",
+            "when": null,
+            "workflow_outputs": [
+                {
+                    "label": "list_of_lists_of_processed_samples",
+                    "output_name": "output",
+                    "uuid": "c331d18c-0574-42aa-8912-cf016a61b8d0"
+                }
+            ]
+        },
+        "3": {
+            "annotation": "",
+            "content_id": "toolshed.g2.bx.psu.edu/repos/bgruening/split_file_to_collection/split_file_to_collection/0.5.0",
+            "errors": null,
+            "id": 3,
+            "input_connections": {
+                "split_parms|input": {
+                    "id": 1,
+                    "output_name": "output"
+                }
+            },
+            "inputs": [
+                {
+                    "description": "runtime parameter for tool Split file",
+                    "name": "split_parms"
+                }
+            ],
+            "label": null,
+            "name": "Split file",
```

Comment:
Could you make the name more explicit? Same for all steps in every workflow.
It is used to populate the "Steps" table on WorkflowHub (e.g. https://workflowhub.eu/workflows/870?version=1)

Reply:
its the tool name, but I can change it according to what its role in the workflow

Reply:
explain for the workflow itself

Reply:
put the tool name in the description of the tool